### PR TITLE
New package: heaptrack-1.1.0

### PR DESCRIPTION
**Pull request by Piraty**

### CODE REVIEW

File: srcpkgs/heaptrack/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pkgname=heaptrack
+version=1.1.0
+revision=1
+build_style=cmake
+# one test fails
+configure_args="-DBUILD_TESTING=OFF"
+makedepends="boost-devel libunwind-devel zlib-devel libdwarf-devel
+ qt5-devel libzstd-devel ecm-devel"
+#gui depends
+hostmakedepends+="qt5-qmake qt5-host-tools extra-cmake-modules kcoreaddons"
```
Comment (xtraeme) [inline srcpkgs/heaptrack/template]: you don't have to append here, just set.

**Comment by Piraty** [comment]
I preferred to state the dependencies for the GUI separately in case the project is split in the future (as comments in upstream/master may suggest)

**Comment by xtraeme** [comment]
@Piraty what I'm saying is that `hostmakedepends` is only declared once. And it's using `+=` rather than `=`

**Comment by Piraty** [comment]
I know. I just wanted to justify the split of the vars, in case that's disliked.

**Pull request closed without merging**